internal/utils/index/index.go

```diff
@@ -93,6 +93,11 @@ func ExtendableOnKonnectExtension[t extensions.ExtendableT](ctx context.Context,
 		obj,
 		KonnectExtensionIndex,
 		func(o client.Object) []string {
+			obj, ok := o.(T)
```

Comment:
Before, we relied here on the passed `obj T` that is always empty - due to that no indexing was happening.

Reply:
The initialization was in raw 97, you moved it upward 🤔

Reply:
It must have been fixed in https://github.com/Kong/gateway-operator/commit/e9fbde9f3d3eaf63b8e968b329d32c2e5b496e0c, I added this comment before rebasing.

Reply:
Oh yes, that's right 👍🏻